contracts/contracts/rem.oracle/src/rem.oracle.cpp

```diff
@@ -84,17 +84,16 @@ namespace eosio {
    std::map<name, vector<double>> oracle::get_relevant_prices() const {
       vector <name> _producers = eosio::get_active_producers();
       std::map<name, vector<double>> prices_data;
+      auto ct = current_time_point();
 
       for (const auto &producer: _producers) {
          auto it = pricedata_tbl.find(producer.value);
          if (it != pricedata_tbl.end()) {
```

Comment:
```suggestion
         if (it != pricedata_tbl.end() && ((ct - it->last_update) < seconds(setprice_window * 2))) {
```
`setprice_window * 2` - to accomodate a valid case when (e.q. `last_update` = 13:01 and `ct` = 14:55)

Reply:
@roman-tik, done.